### PR TITLE
feat(rides): adds ability to see passengers for a ride

### PR DESCRIPTION
**Pull request by codyleyhan**
# Description
- Can now get all passengers for a given ride
screen shot 2018-05-09 at 1 29 48 pm
- Added field when showing rides to list the number of seats taken
screen shot 2018-05-09 at 1 30 03 pm

# Steps to Reproduce
- /api/v1/rides/:id
- /api/v1/rides/:id/passengers

# Tested
- No testing for now

# Approved
- [ ] @jcherianucla 
- [x] @epicchewy

### CODE REVIEW

File: stores/postgres/ride_sql.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ CREATE TABLE IF NOT EXISTS rides (
 
 	rideGetAllSQL = "SELECT * FROM rides WHERE id > $1 LIMIT $2"
 
-	rideGetByIDSQL = "SELECT * FROM rides WHERE id=$1"
+	rideGetByIDSQL = "SELECT *, (SELECT COUNT(*) FROM passengers WHERE ride_id=$1 AND status='accepted') AS seats_taken FROM rides WHERE rides.id=$1;"
```
Comment (epicchewy) [inline stores/postgres/ride_sql.go:31]: where arey ou creating the passengers table?

**Review by epicchewy — CHANGES_REQUESTED** [review]

File: stores/postgres/ride_sql.go (hunk 1 above)
Comment (codyleyhan) [reply stores/postgres/ride_sql.go:31]: Its in the passenger_sql.go file

**Review by epicchewy — APPROVED** [review]

**Comment by epicchewy** [comment]
lgtm

**Comment by codyleyhan** [comment]
Going to start merging with just one reviewer to speed things up.

**Pull request merged**